cmd/operator-sdk/test/local.go

```diff
@@ -82,6 +83,7 @@ func newTestLocalCmd() *cobra.Command {
 		"Use a different operator image from the one specified in the namespaced manifest")
 	testCmd.Flags().StringVar(&tlConfig.localOperatorFlags, "local-operator-flags", "",
 		"The flags that the operator needs (while using --up-local). Example: \"--flag1 value1 --flag2=value2\"")
+	testCmd.Flags().BoolVar(&tlConfig.cleanupOnErrors, "cleanup-on-errors", true, "Clean up created esources even after the test encountered errors")
```

Comment:
```suggestion
	testCmd.Flags().BoolVar(&tlConfig.skipCleanupError, "skip-cleanup-error", false, "Clean up function will be skiped if an error be faced and it be set as true.")
```

Reply:
WDYT about by default it is false and a msg which described more what it will do and when?

Reply:
That could be an option! I just thought it would be safer to propose it this way since it's the behavior that the command already has. Didn't wanna change default behavior.